doc/news/_preparation_next_release.md

```diff
@@ -47,11 +47,9 @@ docker run -it elektra/elektra
 
 The following section lists news about the [plugins](https://www.libelektra.org/plugins/readme) we updated in this release.
 
-### <<Plugin1>>
+### opts
 
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
+- The `opts` plugin now includes deeply nested options and arguments in the generated help message. _(Tobias Schubert @qwepoizt)_
```

Comment:
The library is actually called `opts` and the plugin (which uses the library) `gopts` (because the can't have the same name).

Reply:
Well spotted thank you! I've added a 'g' just now :)